server/src/internalClusterTest/java/org/opensearch/indices/settings/UpdateSettingsIT.java

```diff
@@ -99,6 +100,22 @@ public void testInvalidDynamicUpdate() {
         assertNotEquals(indexMetadata.getSettings().get("index.dummy"), "invalid dynamic value");
     }
 
+    public void testArchivedSettingUpdateOpenIndex() {
```

Comment:
you can rename the test to testArchivedSettingUpdate and cover for both open and close.